### ISSUE TITLE
pac solution import - automatic build

### ISSUE THREAD

**Issue by rajyraman** [microsoft/powerplatform-vscode#322]
Is pac solution import supposed to build the solution zip automatically, or should I always run the dotnet build first before attempting it in a folder with the cdsproj file? It is a bit ambiguous in the help for path param.

![image](https://user-images.githubusercontent.com/5035266/197066692-80393ad9-3e37-4160-8a5c-48a8f7a45a51.png)

When I am running solution import from a folder with cdsproj file I don't have to specify path, but the solution zip is not built either. I think it is better to have the path take the path to cdsproj file, the pac cli automatically building the solution and then importing it. That would make it a 1 step process. Maybe if the full path to the solution zip is specified in the path param, PAC CLI can just import it as well which is the current behaviour. IMHO automatic build would be helpful for people who are using cdsproj.

**Comment by davidjenni** [comment]
@rajyraman no, that is not the intended design, and frankly would invite unintended side effects: `pac solution import` assumes the solution.zip has either been built, or it also might be a result of a prior export from another environment. Trying to build automagically only invites more confusion.

**Comment by rajyraman** [comment]
@davidjenni - The Issue for me is the text `If not specified, assumes the current folder is a cdsproj project` in the path parameter. If the tool sees that the current folder has a cdsproj file it automatically goes to bin/Debug and starts importing the zip file from there. Since I am only operating on the extracted zip contents running `dotnet build` to import the solution into the environment feels like a chore to me. 

There is also the risk that I might have forgotten to run the `dotnet build` after I made some small tweak to the extracted zip contents, or I did a `git pull`, in which case automatically building the zip before import is quite useful. If I just run `pac solution import` the tool knows that I am in the solution folder with cdsproj files, so I think it can also run `dotnet build` for the above-mentioned reasons.

If you have some random zip that you want to import, then pointing the path parameter to that zip feels alright because the zip file is already sitting there.
